src/commands/test/lib/run-tests-command.ts

```diff
@@ -199,4 +205,16 @@ export class RunTestsCommand extends AppCommand {
     let parsedParameters = parseTestParameters(this.testParameters);
     _.merge(manifest.testFramework.data, parsedParameters || {});
   }
+
+  private async processLanguage(): Promise<void> {
+    if (!this.language) {
+      return;
+    }
+    for (let index in this.testParameters) {
+      if (this.testParameters[index].indexOf("language=") === 0) {
+        throw new Error("Argument '--language' cannot be used with a 'language' test parameter");
+      }
+    }
+    this.testParameters.push("language=" + this.language);
```

Comment:
The `language` test parameter appears to be working.  This test run tests the code...

https://mobile.azure.com/users/jonstoneman/apps/xtcsampleios/test/runs/8963bcc9-e123-4a6e-beb1-9e0553cffc34

...and in the output.log for the device...

https://xtc-prod-pipeline.s3-eu-west-1.amazonaws.com/275f880c-23f9-4951-a6b4-a433f7826957/62eb5107-25a4-49b7-be90-0c0680b67d99/files/output.log?AWSAccessKeyId=AKIAI4UZT4FCOF2OTJYQ&Signature=QpwEEg5VFI4SkLd5TKmlMprCVcE%3D&Expires=1577836800

...I see...

```
2017-02-13T17:48:11+01:00 INFO [26208] Invoking pipeline_perform([{"use_device_agent"=>true, "test_parameters"=>{"locale"=>"zh_CN", "language"=>"zh", ...
```